app/src/main/java/com/kamron/pogoiv/OcrHelper.java

```diff
@@ -28,10 +28,10 @@
     private final int heightPixels;
     private final int widthPixels;
     private final boolean candyWordFirst;
-    private final String nidoFemale;
-    private final String nidoMale;
+    private final String nidoFemale = "Nidoran♀";
+    private final String nidoMale = "Nidoran♂";
```

Comment:
Turns out these particular names never get translated apart from Japanese, out of sheer luck... But still there's no need to duplicate these constants. Fixing soon.